ci/openstack/install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source ci/openstack/vars.sh
+if [ "${RUN_OPENSTACK_CI:-}" != "true" ]; then
+    echo RUN_OPENSTACK_CI is set to false, skipping the openstack end to end test.
+    exit
+fi
+
+if [ "${CI_OVER_CAPACITY:-}" == "true" ]; then
+    echo the CI is over capacity, skipping the end-end test.
+    exit 1
+fi
+
+git clone https://github.com/openshift/openshift-ansible ../openshift-ansible
+cd ../openshift-ansible
+git checkout openshift-ansible-3.6.153-1
```

Comment:
Parametrised by what? An environment variable in travis?

For the record, I just wanted to make sure we always use the same version of openshift-ansible so we're not randomly broken by it.

But thinking about it more, maybe we actually always want to use the latest version so we do get notified about breakage and have a chance to file issues/fixes.